infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -63,6 +65,11 @@
      */
     private Map<String, Object> opts;
 
+    /** Handler for this server */
+    @Getter
+    private CorfuMsgHandler handler = new CorfuMsgHandler()
+            .generateHandlers(MethodHandles.lookup(), this);
+
```

Comment:
comments below do not help. 

what is the difference between stream-local address and last issued token?